pagopa/io-app
# [IOCOM-144] rptId calculation extraction from the PnMessageDetails component

## Short description
This PR extracts **rptId** calculating from the `PnMessageDetails` component.

## How to test
Execute the login and open a pn message: if the rptId calculation is correct, you should see the payment notice section correctly
